bin/main.js

```diff
@@ -58,11 +59,16 @@ var load = loader({
     setup: ({cfg}) => new taskcreator.TaskCreator(cfg.taskcluster),
   },
 
+  ses: {
+    requires: ['cfg'],
+    setup: ({cfg}) => new AWS.SES(cfg.aws.ses),
+  },
+
   router: {
-    requires: ['cfg', 'validator', 'Hook', 'taskcreator'],
-    setup: ({cfg, validator, Hook, taskcreator}) => {
+    requires: ['cfg', 'validator', 'Hook', 'taskcreator', 'ses'],
+    setup: ({cfg, validator, Hook, taskcreator, ses}) => {
       return v1.setup({
-        context: {Hook, taskcreator},
+        context: {Hook, taskcreator, ses},
```

Comment:
If you imagine that it should be triggered by the webhook too... (and in future the pulse hook), then I think it's sane to add to the `taskcreator` instead of the `scheduler`.